docs/references/phasing.tex

```diff
@@ -1,111 +1,179 @@
+    \item\textbf{Greenwich sidereal time (GST)}: Angle (typically described in units of hours, where 6 hours is 90$^{\deg}$) that describes the current rotational orientation of the Earth relative to a celestial frame. Greenwich lies on the ``prime meridian'' (i.e., at a longitude of $0^{\deg}$), hence the name. As an example, a source at a right ascension of 14h will appear overhead in Greenwich (at it's highest point in elevation; at either $0^{\deg}$ or $180^{\deg}$ azimuth) at 14h GST. GST has two variants: ``mean'', which where the celestial frame accounts of precession/long-term shifts but no other terms; and ``apparent'', where the smaller/faster terms are included such that the celestial frame is the ``apparent'' frame.
+    \item\textbf{Local sidereal time}: The ``local'' version of the GST, which is simply the sum of the GST and the local longitude. 
+    \item\textbf{Precession}: The slow, steady change of the axis of rotation of the Earth caused chiefly by the Sun and the Moon, whereby in the orbital plane, the rotation axis shifts azimuthally. This causes the axis of rotation (and coordinate frames that are aligned with this axis) to shift by approximately one degree per 72 years, with a cycle of approximately 26,000 years.
+    \item\textbf{North Celestial Pole (NCP)}: The northernmost point on the sky, from the perspective of the reference frame (\emph{typically} aligned close to the axis of rotation for the Earth).
+    \item\textbf{Nutation}: The ``fast'' change of the axis of rotation, dominated by the orbital plane of the moon changing over a period of approximately 19 years (leading to a shift of approximately 9 arcseconds). Multiple small terms ($\sim$ arcsecond magnitude) with short cycles (less than 100 years in length) are grouped into this, typically consisting of hundreds of terms.
+    \item\textbf{Polar motion}: Change in position of the surface of the Earth relative to its rotational axis. This typically produces shifts of a few meters between coordinate frames where the surface of the Earth remains ``fixed'' (e.g., ITRS) and where its defined relative to the ever-changing rotational axis.
+    \item\textbf{Stellar aberration}: Shift in apparent position of celestial sources due to the motion of the observer. This is typically broken into two components -- \emph{annual aberration}, which arises from the Earth's orbinal motion and has a magnitude of $\sim20"$; and \emph{diurnal aberration}, which arises from the rotation of the Earth and has a magnitude of $\sim0.3"$. 
+\end{itemize}
```

Comment:
Can we note that although it's called "stellar" aberration, it applies to all celestial sources including radio galaxies? The name drives me nuts as a radio astronomer 😝

Reply:
Done!